Mathlib/RingTheory/Etale/Basic.lean

```diff
@@ -141,6 +161,7 @@ theorem localization_base [FormallyEtale R Sₘ] : FormallyEtale Rₘ Sₘ :=
     ⟨FormallyUnramified.localization_base M, FormallySmooth.localization_base M⟩
 #align algebra.formally_etale.localization_base Algebra.FormallyEtale.localization_base
 
+/-- The localization of a formally etale map is formally etale. -/
```

Comment:
```suggestion
/-- The localization of a formally étale map is formally etale. -/
```

Reply:
We had a discussion about this during the session, and I pointed out that searching for adèle in Lean was a pain for me because I didn't have an è on my keyboard. People on the call whose first language contained accents immediately told me how to do it on Ubuntu (Alt-Gr # e) and now look at me I'm writing accents in github comments, but do you think there's an argument @semorrison for de-accenting words like this? Alternatively, should the Lean declaration be `FiniteAdèleRing`?

Reply:
I never put the accent on the "a" in `TuranGraph`/`IsTuranMaximal` in #9317, but (knowing that you can type any character in Ubuntu using Ctrl-Shift-U then the Unicode hex codepoint) put the accent everywhere _in the commentary_. The Lean declaration should be `FiniteAdeleRing`.

Reply:
OK Michael Stoll on the Zulip has convinced me you're right.